command_line_rock_paper_scissors.py

```diff
@@ -1,10 +1,14 @@
 """Object-Oriented Rock Paper Scissors"""
 import random
 
+import sys
```

Comment:
I noticed this because of the blank line before `import sys`, but this comment actually applies to all import statements...

All imports should be done in "blocks", each block separated by a blank line, starting with builtin Python packages, and each following block becoming more and more "local" to the current module. In most cases this can be accomplished with 2 or 3 blocks:
1. Builtins
2. 3rd-party (installed/"contrib") packages
3. Current codebase

Additionally, note that all imports within a block should be in alphabetical order.

An example of using all three:
```python
import abc
from random import randrange
from os.path import abspath, exists
import sys

from falcon import Falcon
import requests
from sqlalchemy.orm import RelationshipProperty

from my_app import exc
from my_app.somewhere import something


def some_function():
```

In the case of this application, there are (currently) no contrib packages, so you can skip the middle bock.